src/main/java/com/coherent/task/utils/common/CommonUtils.java

```diff
@@ -0,0 +1,31 @@
+package com.coherent.task.utils.common;
+
+import com.coherent.task.utils.logger.LogManager;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Properties;
+
+public class CommonUtils extends LogManager {

```

Comment:
Could you create a private constructor for this class for preventing to create instances of this class?